src/Util/Constants.cs

```diff
@@ -0,0 +1,28 @@
+﻿/*
+ * Copyright (c) 2000 - 2020 The Legion of the Bouncy Castle Inc. (https://www.bouncycastle.org) 
```

Comment:
Is the copyright here correct?

Reply:
Whoops, no that isn't right. I fixed that.